libs/model/src/models/thread_subscriptions.ts

```diff
@@ -3,10 +3,17 @@ import Sequelize from 'sequelize'; // must use "* as" to avoid scope errors
 import { z } from 'zod';
 import type { ModelInstance } from './types';
 
-export type ThreadSubscriptionAttributes = z.infer<typeof ThreadSubscription>;
+export type ThreadSubscriptionAttributes = z.infer<
+  typeof ThreadSubscription
+> & {
+  // associations
+  Thread?: any;
+};
 
 export type ThreadSubscriptionInstance =
-  ModelInstance<ThreadSubscriptionAttributes>;
+  ModelInstance<ThreadSubscriptionAttributes> & {
+    // no mixins used
+  };
```

Comment:
do we need this?

Reply:
Yes... I guess.  It's part of the convention on the backend. We're using these in multiple places and I think the point is that we can add these in the future.